applications/luci-app-statistics/root/etc/config/luci_statistics

```diff
@@ -49,6 +49,11 @@ config statistics 'collectd_unixsock'
 
 # input plugins
 
+config statistics 'collectd_apcups'
+	option enable '1'
```

Comment:
This should be 0 in the default config. 
If the value is 1 and the apcups plugin is not installed, collectd will not start at all. (So, your current default value 1 would effective disable all statistics for everyone who has not installed the apcups plugin.)